gpkit/nomials/data.py

```diff
@@ -66,27 +72,6 @@ def init_from_nomials(self, nomials):
     def __repr__(self):
         return "gpkit.%s(%s)" % (self.__class__.__name__, hash(self))
 
-    def _get_varkey(self, var):
-        """Cast a var associated with this problem to type VarKey
-
-        Arguments
-        ---------
-        var (Variable, VarKey, or str):
-            the variable to cast
-
-        Returns
-        -------
-        VarKey
-        """
-        if isinstance(var, VarKey):
-            return var
-        if var in self.varstrs:
-            return self.varstrs[var]
-        try:
-            return var.key
-        except AttributeError:
-            raise TypeError("Cannot convert %s to VarKey" % var)
-
     def sub(self, substitutions, val=None, require_positive=True):
         if hasattr(self, "nomials"):
             subbed_nomials = [n.sub(substitutions, val, require_positive)
```

Comment:
down below (not included in the diff), `substitution` is called -- but it isn't defined in the file.